src/scp/QuorumSetUtils.cpp

```diff
@@ -46,7 +46,7 @@ QuorumSetSanityChecker::QuorumSetSanityChecker(SCPQuorumSet const& qSet,
 bool
 QuorumSetSanityChecker::checkSanity(SCPQuorumSet const& qSet, int depth)
 {
-    if (depth > 3)
+    if (depth > 4)
```

Comment:
I would split this (increase max depth) into a different commit

Reply:
Hmm seems like this change should be grouped with the others since it's required and directly related to the quality. Why do you think it should be split?

Reply:
we can increase this without adding the notion of "CRITICAL" validator. I am not saying to put this in a different PR, just a different commit so that each commit is better scoped.